Packs/DomainTools_Iris/Scripts/DomainToolsAddDomainRiskScoreToContext/add_domain_riskscore_to_context.yml

```diff
@@ -0,0 +1,18 @@
+commonfields:
+  id: AddDomainRiskScoreContext
+  version: -1
+name: AddDomainRiskScoreToContext
+script: '-'
+type: python
+tags:
+- DomainTools
+enabled: true
+args:
+- name: domaintools_data
+  required: true
+outputs:
+- contextPath: HighRiskPivotedDomains.Name
+  description: The domain name
+- contextPath: HighRiskPivotedDomains.OverallRiskScore
```

Comment:
If changing the `output_prefix` as I suggested, make sure to update the `contextPath` here as well.

Reply:
updated!